Shared/Samples/Show service area/ShowServiceAreaView.swift

```diff
@@ -0,0 +1,227 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct ShowServiceAreaView: View {
+    // MARK: - View
+    
+    /// The currently selected graphic type.
+    ///
+    /// Used to track whether to add facilities or barriers to the map.
+    @State private var selectedGraphicType: GraphicType = .facility
+    /// The error shown in the error alert.
+    @State private var error: Error?
+    /// First time break property set in first stepper.
+    @State private var firstTimeBreak: Int = 3
+    /// Second time break property set in second stepper.
+    @State private var secondTimeBreak: Int = 8
+    
+    /// The data model for the sample.
+    @StateObject private var model = Model()
+    
+    var body: some View {
+        MapView(map: model.map, graphicsOverlays: model.graphicsOverlays)
+            .onSingleTapGesture { _, point in
+                switch selectedGraphicType {
+                case .facility:
+                    model.addFacilityGraphic(at: point)
+                case .barrier:
+                    model.addBarrierGraphic(at: point)
+                }
+            }
+            .toolbar {
+                ToolbarItemGroup(placement: .bottomBar) {
+                    Picker("Mode", selection: $selectedGraphicType) {
+                        ForEach(GraphicType.allCases, id: \.self) {
+                            Text($0.label)
+                        }
+                    }
+                    .pickerStyle(.segmented)
+                    Spacer()
+                    Menu {
+                        Stepper("Second: \(secondTimeBreak)", value: $secondTimeBreak, in: 1...15)
+                        Stepper("First: \(firstTimeBreak)", value: $firstTimeBreak, in: 1...15)
+                    } label: {
+                        Label("Time", systemImage: "gear")
+                    }
+                    Spacer()
+                    Button("Service Area") {
+                        Task {
+                            do {
+                                try await model.showServiceArea(timeBreaks: [Double(firstTimeBreak), Double(secondTimeBreak)])
+                            } catch {
+                                self.error = error
+                            }
+                        }
+                    }
+                    Spacer()
+                    Button("Clear", systemImage: "trash.fill") {
```

Comment:
```suggestion
                    Button("Clear", systemImage: "trash") {
```
To be consistent with other trash buttons in the app. Not a must to update the screenshot.